docs/psql.md

```diff
@@ -0,0 +1,52 @@
+# Google Cloud Spanner PostgreSQL Adapter - PSQL Support
+
+PGAdapter supports [psql](https://www.postgresql.org/docs/current/app-psql.html) versions 11, 12, 13 and 14.
+
+## Usage
+
+First start PGAdapter in `psql` mode (see [README](../README.md) for more possibilities on how to run
+PGAdapter).
+
+```shell
+export GOOGLE_APPLICATION_CREDENTIALS=/path/to/credentials.json
+docker pull us-west1-docker.pkg.dev/cloud-spanner-pg-adapter/pgadapter-docker-images/pgadapter
+docker run \
+  -d -p 5432:5432 \
+  -v ${GOOGLE_APPLICATION_CREDENTIALS}:${GOOGLE_APPLICATION_CREDENTIALS}:ro \
+  -e GOOGLE_APPLICATION_CREDENTIALS \
+  us-west1-docker.pkg.dev/cloud-spanner-pg-adapter/pgadapter-docker-images/pgadapter \
+  -p my-project -i my-instance -d my-database \
+  -x -q
+```
+
+Note:
```

Comment:
favor of de-duping wherever possible

Reply:
See above. I think it's better to have this here to be able to provide a full example of starting PGAdapter in psql-mode.